bundles/AdminBundle/Resources/public/js/pimcore/object/tags/advancedManyToManyObjectRelation.js

```diff
@@ -230,6 +230,36 @@ pimcore.object.tags.advancedManyToManyObjectRelation = Class.create(pimcore.obje
 
 
         if (!readOnly) {
+            listeners.push({'afterrender':
```

Comment:
this is the wrong place for that, isn't it? 
because it modifies the listeners array, that is created, populated and assigned to the column config in the for-loop above. 
so, this event listener will only be added to the last column, not to all!